schema/rpc.fbs

```diff
@@ -34,7 +34,10 @@ union RpcMessage {
     AutoBoneEpochResponse,
     OverlayDisplayModeRequest,
     OverlayDisplayModeChangeRequest,
-    OverlayDisplayModeResponse
+    OverlayDisplayModeResponse,
+    SerialTrackerRebootRequest,
```

Comment:
I think that referring to Serial here is strange - why does the GUI care if the tracker is connected via serial or not? Perhaps it should just be a `RebootRequest`?

Can you clarify why Serial is specified here as opposed to a more generic command?

Reply:
This commands are only available in the Serial Monitor Window. So that is why i did Add this Serial over it. Because the Media it will be travel over is the USB Serial connection of the SerialMonitor. 

There is currently no Serial Monitor on the Network interface on the Tracker Firmware. The Firmware would need a major redesign to be able to have something close like that. (If it means that the tracker have to first establish the connection, and then init all the sensors when the monitor connection is up, but that would lead to different issues).